aspnetcore/mvc/controllers/routing.md

```diff
@@ -3,65 +3,56 @@ title: Routing to controller actions in ASP.NET Core
 author: rick-anderson
 description: Learn how ASP.NET Core MVC uses Routing Middleware to match URLs of incoming requests and map them to actions.
 ms.author: riande
-ms.date: 12/05/2019
+ms.date: 1/25/2020
```

Comment:
Something has gone awry with the sections, or delimiters. I'm seeing some sections duplicated. 

Also I expected to see the `IActionConstraint` section gone. We don't want to encourage people to use it.

![image](https://user-images.githubusercontent.com/1430011/74097527-b429c000-4ac1-11ea-8500-16f9775c5ba0.png)